pkgs/applications/audio/deadbeef/plugins/musical-spectrum.nix

```diff
@@ -0,0 +1,38 @@
+{ deadbeef
+, fetchFromGitHub
+, fftw
+, glib
+, gtk3
+, lib
+, pkg-config
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "deadbeef-musical-spectrum-plugin";
+  version = "0.9";
```

Comment:
According to https://nixos.org/nixpkgs/manual/#sec-package-naming, the `version` should be `unstable-yyyy-mm-dd`, when using git snapshots.

Reply:
I changed the version to `unstable-2020-07-01`.